pkg/health/client.go

```diff
@@ -0,0 +1,40 @@
+package health
```

Comment:
As discussed, this should be a diagnostics package because the endpoint is exposed from the DC/OS diagnostics component.